builtin/game/chatcommands.lua

```diff
@@ -182,14 +182,16 @@ core.register_chatcommand("grant", {
 		local privs = core.get_player_privs(grantname)
 		local privs_unknown = ""
 		for priv, _ in pairs(grantprivs) do
-			if priv ~= "interact" and priv ~= "shout" and
-					not core.check_player_privs(name, {privs=true}) then
-				return false, "Your privileges are insufficient."
-			end
-			if not core.registered_privileges[priv] then
+			local def = core.registered_privileges[priv]
+			if def then
+				if not def.basic_priv and
+						not core.check_player_privs(name, {privs=true}) then
```

Comment:
No it doesn't. Also, this is consistent with the rest of the file, and is more readable.
Also:

> Note that these are only guidelines for more readable code. In some (rare) cases they may result in less readable code. Use your best judgement. 

http://dev.minetest.net/Lua_code_style_guidelines